types/jasmine-given/index.d.ts

```diff
@@ -3,9 +3,17 @@
 // Definitions by: Shai Reznik <https://github.com/shairez>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function Given(func: (done?: () => void) => void): void;
-declare function When(func: (done?: () => void) => void): void;
-declare function Then(func: (done?: () => void) => void): void;
-declare function Then(label: string, func: (done?: () => void) => void): void;
-declare function And(func: (done?: () => void) => void): void;
-declare function Invariant(func: (done?: () => void) => void): void;
+/** Action method that should be called when the async work is complete */
+interface DoneFn extends Function {
```

Comment:
You don't need extends Function clause. Given you have call signature should suffice.

Reply:
changed